web/src/user/LibraryPage/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { LayoutType } from "@goauthentik/common/ui/config";
+
+import type { Application } from "@goauthentik/api";
+
+export type AppGroupEntry = [string, Application[]];
+export type AppGroupList = AppGroupEntry[];
+
+export type PageUIConfig = {
```

Comment:
Yeah thats a fair point, allthough maybe it would be better to go the other way around and compose `UIConfig` of smaller interfaces, and we don't have to add things in two places, but that can be a later PR if we want to do that